docs/apps/uaa/cloudsql.tf

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "google_compute_global_address" "uaa_db_instance_ip" {
+  provider = "google-beta"
+
+  name          = "${var.uaa_db_instance_name}-ip"
+  purpose       = "VPC_PEERING"
+  address_type = "INTERNAL"
+  prefix_length = 16
+  network       = "${data.google_compute_network.default.self_link}"
+}
+
+resource "google_service_networking_connection" "uaa_db_instance_private_vpc_connection" {
+  provider = "google-beta"
+
+  network       = "${data.google_compute_network.default.self_link}"
+  service       = "servicenetworking.googleapis.com"
+  reserved_peering_ranges = ["${google_compute_global_address.uaa_db_instance_ip.name}"]
+}
+
+resource "google_sql_database_instance" "uaa" {
+
+  name = "${var.uaa_db_instance_name}"
+  region = "${data.google_compute_subnetwork.default.region}"
+
+  database_version = "MYSQL_5_7"
+
+  depends_on = [
+    "google_service_networking_connection.uaa_db_instance_private_vpc_connection"
+  ]
+
```

Comment:
I think if this is the given setup, the DB should be on a decent size machine with regional availability; I'd rather the instructions be safe by default with notes to downsize rather than the other way around

Reply:
Updated to use `REGIONAL` for HA and enabled backups and binary logs